lib/matplotlib/image.py

```diff
@@ -667,6 +667,13 @@ def set_data(self, A):
         ----------
         A : array-like
```

Comment:
Could add here that this now also accepts a `PIL.Image.Image` type?

Reply:
Technically, PIL images *are* array-likes (in the sense that `asarray`, for example, works on them).

Reply:
OTOH not every place that takes array-likes can take a PIL image. Maybe we need a definition of *array-like* (technically https://stackoverflow.com/questions/40378427/numpy-formal-definition-of-array-like-objects).

However, from a practical point of view, most places we use *array-like* do not reasonably take PIL images (even if they sometimes technically could). For the few places of dedicated image-related stuff, I would be explicit and use `array-like or PIL.Image.Image`. Not every user will know that an PIL Image is technically an array-like).

Reply:
fair enough, edited accordingly